### PR TITLE
Add dependency on jersery-servlet.

### PR DESCRIPTION
**Pull request by anwar6953**
Fixes issue:
http://builds.cask.co/browse/CG-RCSCP-34/log

image

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -138,6 +138,11 @@
       <version>${scala.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.sun.jersey</groupId>
+      <artifactId>jersey-servlet</artifactId>
+      <version>1.18</version>
```
Comment (chtyim) [inline pom.xml:144]: Should be of test scope only.

Comment (anwar6953) [inline pom.xml:144]: Will make that change. This was recently excluded in CDAP 3.3.0, right?

Comment (chtyim) [inline pom.xml:144]: Ideally it should be part of cdap-unit-test. Better fix it from the cdap side.

Comment (anwar6953) [inline pom.xml:144]: Ok; https://github.com/caskdata/cdap/pull/5030

**Pull request closed without merging**